### DISCUSSION TITLE
ToMe & CFG Dynamic Thresholding

### DISCUSSION THREAD

**Discussion by KaruroChori** [Comfy-Org/ComfyUI#800] [Q&A] [answered]
Hi all,
Does anyone know if these or similar features are supported either in core ComfyUI or via extensions?
- [ToMe](https://github.com/dbolya/tomesd) for faster generation via token merge.
- [CFG Dynamic Thresholding](https://github.com/mcmonkeyprojects/sd-dynamic-thresholding) to avoid saturation issues in the image.

**Comment by comfyanonymous** [comment, marked as answer]
ToMe is: _for_testing->TomePatchModel

For dynamic thresholding it's not the same but it's similar, see the tonemap node in: https://github.com/comfyanonymous/ComfyUI_experiments

**Reply by KaruroChori** [reply]
Thank you!
